linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/pwquality_conf_overriden.fail.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# packages = authselect
+# platform = Oracle Linux 8,Oracle Linux 9,Red Hat Enterprise Linux 8,Red Hat Enterprise Linux 9
+# variables = var_password_pam_retry=3
+
+source common.sh
+
+echo "retry = 3" >  /etc/security/pwquality.conf
+
+for file in ${configuration_files[@]}; do
+	{{{ bash_ensure_pam_module_options('/etc/authselect/custom/testingProfile/$file',
```

Comment:
I know this macro is still used somewhere but it should be gradually moved to the new macros, which are refactored and aligned between Ansible and Bash. Could you replace the `bash_ensure_pam_module_options` by `bash_ensure_pam_module_option`, please?